tasks/managed/prepare-fbc-snapshot/prepare-fbc-snapshot.yaml

```diff
@@ -0,0 +1,394 @@
+---
```

Comment:
This task is largely the same as `tasks/managed/prepare-fbc-release/prepare-fbc-release.yaml` which was erroneously deleted in https://github.com/konflux-ci/release-service-catalog/pull/1423/files#diff-f2104c84d5e2c6a96df61bdd652633cf772566bfbf4e48ae2da515b01e90fd3a

The primary difference is how the tag is defined.